tests/ui/specs/Scroller/EditableScroller/Scroller-QWT-4513-specs.js

```diff
@@ -0,0 +1,39 @@
+const ScrollerPage = require('../ScrollerPage');
+const {expectFocusedItem} = require('../Scroller-utils');
+
+describe('Editable Scroller', function () {
+	beforeEach(async function () {
+		await ScrollerPage.open('EditableItem');
+	});
+
+	it('Should change item position with editableCentered', async function () {
```

Comment:
QWT-4513 also check center position .
([Enact QA Sampler] sandstone/Scroller>with editable items: Items horizontally center align in Scroller with 'editableCentered')

Could you please add another tc ("it")

Reply:
Step 3~ Step4 are worked in screenshot tests. I added comment about QWT-4513. Please check this.